src/test/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorSimulatorTest.java

```diff
@@ -2,6 +2,8 @@
 
 import java.util.List;
 
+import org.cloudfoundry.client.v2.domains.DomainResource;
+import org.cloudfoundry.client.v2.organizations.ListOrganizationDomainsResponse;
```

Comment:
Looks superfluous to me.

Would be good, though, if we could have a little coverage on the `CFAccessorSimulator` though.

Reply:
I added a test to cover the added domain method.